flipdisc/framework/common.py

```diff
@@ -18,3 +19,40 @@ class REDIS_KEYS(enum.Enum):
 HDMI_INPUT_STREAM = "HDMI_STREAM"
 
 OUTPUT_STREAM = "OUT_STREAM"
+
+
+class ScheduledFunction:
+    def __init__(self, function, delay, periodic, loop, *args):
```

Comment:
Rather than absorbing all extra constructor arguments, I think it would be a cleaner API to take an iterable here. This would allow the function and args to be consecutive arguments to the constructor, and makes it easier to make the other parameters keyword args with default values.

```suggestion
    def __init__(self, function: Callable, args: Iterable, delay: float, periodic: bool=False, loop=None):
```